pkg/api/controller.go

```diff
@@ -3016,7 +3022,7 @@ func (c *Controller) handleAPIErrorCallback(ctx context.Context, w http.Response
 		cb(w, r, http.StatusLocked, "Too many attempts, try again later")
 	case errors.Is(err, kv.ErrSlowDown):
 		log.Debug("KV Throttling")
-		cb(w, r, http.StatusServiceUnavailable, "Throughput exceeded. Slow down and retry")
+		cb(w, r, http.StatusServiceUnavailable, "Key value throughput exceeded. Slow down and retry")
```

Comment:
Though the edited error message is better, even changing the body of an error response is considered a breaking change to our 1.0 guarantees, so we need to revert this one.